scos_usrp/hardware/tests/test_calibration.py

```diff
@@ -1,574 +0,0 @@
-"""Test aspects of ScaleFactors."""
```

Comment:
With this deleted (now existing in [`scos-actions`](https://github.com/NTIA/scos-actions/blob/master/scos_actions/tests/test_calibration.py), should a new test be added analogous to [this](https://github.com/NTIA/scos-tekrsa/blob/y_factor_cal_complete/scos_tekrsa/tests/test_calibration.py)?